web/portal/brand/src/translations/en.json

```
@@ -104,8 +104,8 @@
   "Charge period*: Charge period": "Charge period*: Charge period",
   "Clid": "Clid",
   "Client": "Client",
-  "Client Portal_one": "Client Portal",
-  "Client Portal_other": "Client Portals",
+  "Client Portal_one": "Administration Portal",
```

Don't keep translation keys if they no longer make sense
```suggestion
  "Administrator Portal_one": "Administration Portal",
```